src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -395,6 +395,15 @@ EOF
   @Override
   def setupToolVersion() {
     super.setupToolVersion()
+    // Java required on nodejs pipeline, if data import only available as java job, but project itself is nodejs
+    def statusCodeJava21 = steps.sh script: 'grep -F "JavaLanguageVersion.of(21)" aat/build.gradle', returnStatus: true
```

Comment:
It is not necessarily lives in "aat/build.gradle" location, example here https://github.com/hmcts/civil-ccd-definition , build.gradle is at the root of project, not under "aat" subdirectory.

Reply:
Is that build also using a java job on nodejs?

Reply:
[2cd0362](https://github.com/hmcts/cnp-jenkins-library/pull/1280/commits/2cd0362c3eeac7aa3b6a08c82d0111cc8ba5b0f8) should be checking any that have 21

Reply:
Yes, that build in my understanding is similar that is nodejs using java job. After looking in more repos (there aren't many, maybe 5), I'd say that `build.gradle` more likely to be at the root level, and not under aat subdirectory.